plugins/inputs/vsphere/endpoint.go

```diff
@@ -437,7 +437,7 @@ func (e *Endpoint) discover(ctx context.Context) error {
 			}
 
 			// Fill in datacenter names where available (no need to do it for Datacenters)
-			if res.name != "Datacenter" {
+			if res.name != "datacenter" {
```

Comment:
Is there any value in making this comparison case insensitive, or is it always capitalized?

Reply:
It’s always capitalized. It’s just a key used internally in the code.